src/span-data.ts

```diff
@@ -236,10 +277,9 @@ export const UNCORRELATED_CHILD_SPAN = createPhantomSpanData(
 
 /**
  * A virtual trace span that indicates that a real child span couldn't be
- * created because the corresponding root span was disallowed by user
- * configuration.
+ * created because the Trace Agent was disabled.
  */
-export const UNTRACED_CHILD_SPAN = createPhantomSpanData(SpanType.UNTRACED);
+export const DISABLED_CHILD_SPAN = createPhantomSpanData(SpanType.UNTRACED);
```

Comment:
This is going to sound nit-picky...

Is there a reason why the concept was renamed on the left hand side but left the same on the right hand side?

Based on your decision to rename the export, it seems that you prefer external users to conceptually think about this concept as `disabled`. If that is the right conceptualization, why is not the right conceptualization for the internals in this file?

Reply:
The variable formerly named `UNTRACED_CHILD_SPAN` is now only used when the Trace Agent is disabled which is why the LHS changed (where as before it is used for a disabled agent or for unsampled traces). As for the RHS, there continues to be multiple reasons for a span object to represent "untraced" work -- either unsampled, or disabled.

I could get behind the idea that we could just introduce `UNSAMPLED` and `DISABLED` in place of `UNTRACED`, though.

Reply:
+1 on the rename. I won't block this PR, but conceptual clarity is important.